components/tidb_query_expr/src/lib.rs

```diff
@@ -933,6 +933,7 @@ fn map_expr_node_to_rpn_func(expr: &Expr) -> Result<RpnFnMeta> {
         ScalarFuncSig::SubDateDurationDecimalDatetime => sub_date_time_duration_interval_any_as_datetime_fn_meta::<Decimal>(),
         ScalarFuncSig::FromUnixTime1Arg => from_unixtime_1_arg_fn_meta(),
         ScalarFuncSig::FromUnixTime2Arg => from_unixtime_2_arg_fn_meta(),
+        ScalarFuncSig::TimestampDiff => timestamp_diff_fn_meta(),
```

Comment:
```suggestion
        ScalarFuncSig::TimeStampDiff => timestamp_diff_fn_meta(),
```

Reply:
`timestamp` is a word. BTW, The letter case is determined by `tipb` so it can not simply changed.

Reply:
Got.